src/layer/marker/Icon.Default.js

```diff
@@ -57,6 +57,10 @@ export var IconDefault = Icon.extend({
 		           DomUtil.getStyle(el, 'backgroundImage');	// IE8
 
 		document.body.removeChild(el);
-		return this._stripUrl(path) || '';
+		path = this._stripUrl(path);
+		if (path) { return path; }
+		var link = document.querySelector('link[href$="leaflet.css"]');
```

Comment:
I guess this should be hoisted up, so if this runs successfully the fallback method doesn't run?

Reply:
I'm not sure: css-method can be used not only to detect path, it also allows to *override* path.